dot/state/block.go

```diff
@@ -57,11 +59,12 @@ func NewBlockState(db chaindb.Database, bt *blocktree.BlockTree) (*BlockState, e
 	}
 
 	bs := &BlockState{
-		bt:        bt,
-		baseDB:    db,
-		db:        chaindb.NewTable(db, blockPrefix),
-		imported:  make(map[byte]chan<- *types.Block),
-		finalized: make(map[byte]chan<- *types.Header),
+		bt:         bt,
+		baseDB:     db,
+		db:         chaindb.NewTable(db, blockPrefix),
+		imported:   make(map[byte]chan<- *types.Block),
+		finalized:  make(map[byte]chan<- *types.Header),
+		pruneKeyCh: make(chan []common.Hash, 1000),
```

Comment:
reason for 1000? seems very large

Reply:
`pruneKey()` Go-routine might take time to process the pruning in storage and I didn't want the `BlockState` to wait for the channel. I wasn't sure of how many keys could be pruned at one go. Can you suggest a fair value?